install-php-extensions

```diff
@@ -1479,12 +1496,14 @@ fi
 
 sortModulesToInstall
 
+docker-php-source extract
```

Comment:
Why did you move these line up?

Reply:
Because i use `$BUNDLED_MODULES` in `installPickle` to check if pickle will be required - its only installed when at least one of `$PHP_MODULES_TO_INSTALL` is not a bundled module.

Reply:
Yep, sure, `installPickle` must be after `docker-php-source extract` and `BUNDLED_MODULES=...`,
I was asking if there's a reason to move those 2 (or 3 considering `installPickle`) lines up

Reply:
Yes- because pickle requires the `zip` module ( the line `PHP_MODULES_TO_INSTALL="zip $PHP_MODULES_TO_INSTALL"` in `installPickle`) which requires `apt`/`apk` packages to be installed - so this section needs to happen before the platform packages are calculated and installed.

Reply:
Which actually, I'm looking - ill change this to check if zip is already installed or in the list.